includes/views/voltron.txt

```diff
@@ -0,0 +1,66 @@
+<pre id="voltron">
```

Comment:
The pre tags should be in the including file, this should just be plain text.

Reply:
This came after I rebased after https://github.com/aspirepress/AspireUpdate/commit/f6e835df49ffedb9aa7769b57fc170af2c0088aa but the `translation` branch wasn't updated.